package.json

```diff
@@ -21,5 +21,8 @@
     "eslint-plugin-mozilla": "0.0.3",
     "eslint-plugin-react": "^5.2.2"
   },
-  "repository": "https://github.com/mozilla/addon-recommendation-shield-study.git"
+  "repository": "https://github.com/mozilla/addon-recommendation-shield-study.git",
+  "dependencies": {
+    "shield-studies-addon-utils": "^1.2.0"
```

Comment:
You probably want to pin this to the exact version. We've run into issues where a seemingly version-compatible release has broken things in our projects, so we prefer to not match compatible versions (see [the dependencies docs](https://docs.npmjs.com/files/package.json#dependencies) for info on what `^1.2.0` does).

In other words, change this to `"1.2.0"`.